react/ProductSearchContextProvider.js

```diff
@@ -1,68 +1,110 @@
 import React, { Component } from 'react'
-import { path } from 'ramda'
 
 import DataLayerApolloWrapper from './components/DataLayerApolloWrapper'
-import SearchQueryContainer from './components/SearchQueryContainer'
-import { searchContextPropTypes } from './constants/propTypes'
-import { SearchQueryContext } from './constants/searchContext'
 
-class ProductSearchContextProvider extends Component {
-  static propTypes = searchContextPropTypes
+import { processSearchContextProps } from './helpers/searchHelpers'
+import searchQuery from './queries/searchQuery.gql'
+import { Query } from 'react-apollo'
```

Comment:
absolute imports should come before relative ones :pick:

Reply:
fixed